plugins/gitignore/gitignore.plugin.zsh

```diff
@@ -1,4 +1,4 @@
-function gi() { curl -sL https://www.gitignore.io/api/$@ ;}
+function gi() { curl -sL https://www.gitignore.io/api/`echo $@ | sed s/" "/,/g` ;}
```

Comment:
Check out the [14.3. Parameter Expansion](http://zsh.sourceforge.net/Doc/Release/Expansion.html#Parameter-Expansion) section in the ZSH reference manual if you want to learn more about it. It'll show you tons of stuff you can do like this, possibly replacing a bunch of external calls to `grep`, `sed`, `tr`, and the like.